# Copyright 2026 The OpenXLA Authors.
#
# Licensed under the Apache License, Version 2.0 (the "License");
# you may not use this file except in compliance with the License.
# You may obtain a copy of the License at
#
#     http://www.apache.org/licenses/LICENSE-2.0
#
# Unless required by applicable law or agreed to in writing, software
# distributed under the License is distributed on an "AS IS" BASIS,
# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
# See the License for the specific language governing permissions and
# limitations under the License.
# ==============================================================================
We don't build triton's native amd backend in Triton. We could set this up,
but as it's only one test, I've disabled it for now. Any tests that run on AMD,
would not be able to get run on presubmit anyway.

--- a/python/test/unit/language/test_compile_only.py
+++ b/python/test/unit/language/test_compile_only.py
@@ -333,6 +333,8 @@ def test_signature_ordering():
     triton.compile(src=src, target=target)
 
 
+import pytest
+@pytest.mark.skip("we don't build triton's native amd backend")
 def test_fp8_compiles_for_multiple_architectures_hip():
     """
     Validate FP8 compilation succeeds for architectures with different

